src/features/sandbox-banner/__tests__/SandboxBanner.stories.test.tsx

```diff
@@ -1,3 +1,4 @@
+jest.retryTimes(3);
```

Comment:
Why do we have this line?

Reply:
retry tries the test suites in this file because this is known to be flaky, because of timing